package com.revature.team_manager.util.exceptions;

public class DataSourceException extends RuntimeException{
    public DataSourceException(String message, Throwable cause){ super(message, cause); }
}
